Sources/tree_sitter_language_resources/LanguageResources/go/structure.scm

```diff
@@ -0,0 +1,175 @@
+(import_declaration
```

Comment:
what are these files for?

Reply:
I should have explained originally.

But, it was me that added them to go and gomod. They are used to get information about the syntactic structure of the code, which I use for indentation calculations. They are also handy for finding matching elements, like `if`-`{`-`}`. They aren't *technically* part of this change, but got pulled in as part of rebuilding the framework from the parser sources.

Reply:
thanks. nice!